src/components/TransactionDetails/index.js

```diff
@@ -511,7 +511,21 @@ export default function TransactionDetails(props) {
     props.setcustomSlippageError('valid`')
   }
 
-  const checkBounds = slippageValue => {
+  const [initialSlippage] = useState(props.rawSlippage)
```

Comment:
no need to wrap `props.rawSlippage` in a stateful wrapper, you can just use it directly in the `useEffect` below!

Reply:
Yep this makes sense - its weird if use just the props im able to break the input field - I'll see why this happening, might be some delay when rawSliipage gets set in parent component